src/utils/prices.ts

```diff
@@ -82,10 +110,13 @@ export function computeSlippageAdjustedAmounts(
   trade: Trade | undefined,
   allowedSlippage: number
 ): { [field in Field]?: CurrencyAmount } {
+  // @ts-ignoe
   const pct = basisPointsToPercent(allowedSlippage)
+  if (pct) {
+  }
```

Comment:
Should we be doing something with `pct` value? Because this `if` does nothing and we can delete this 4 lines.